src/GovUk.Education.ExploreEducationStatistics.Common/Model/Data/Query/ObservationQueryContext.cs

```diff
@@ -10,27 +9,13 @@ public class ObservationQueryContext
         public Guid SubjectId { get; set; }
         public TimePeriodQuery TimePeriod { get; set; }
         public IEnumerable<Guid> Filters { get; set; }
+        // TODO EES-3328 - remove BoundaryLevel from ObservationQueryContext as we now store it on
```

Comment:
Just wondering if it's better to use the `obsolete` attribute in order to make TODOs more obvious in the logs i.e. 
```
[obsolete("TODO EES-3319 - remove backwards-compatibility for Map Configuration without its
                    // own Boundary Level selection")]
```

Reply:
I could do, but it's not always in places where I need to add attributes that I need to comment up.  In the ticket description though for EES-3319 I have said to look for this specific TODO text

Reply:
Cool, sounds good